Bugsnag/Payload/BugsnagEvent.m

```diff
@@ -680,6 +687,28 @@ - (void)setUnhandled:(BOOL)unhandled {
     self.handledState.unhandled = unhandled;
 }
 
+// MARK: - <BugsnagFeatureFlagStore>
+
+- (void)addFeatureFlagWithName:(NSString *)name variant:(nullable NSString *)variant {
+    BSGFeatureFlagStoreAddFeatureFlag(self.featureFlagStore, name, variant);
```

Comment:
Why do these need to be C functions?

Reply:
They don't need to be, but C functions take up less space in the binary.

Reimplementing these in a `BSGFeatureFlagStore` Objective-C class adds an additional 2kB.